src/ontology/ro-edit.owl

```diff
@@ -5646,6 +5648,18 @@ AnnotationAssertion(oboInOwl:creation_date obo:RO_0004047 "2018-03-13T23:55:19Z"
 AnnotationAssertion(rdfs:label obo:RO_0004047 "causally upstream of or within, positive effect")
 SubObjectPropertyOf(obo:RO_0004047 obo:RO_0002418)
 
+# Object Property: obo:RO_0007000 (has_driver)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0007000 "A relation between a material entity and a system, in which the material entity is any natural or human-influenced factor that directly or indirectly causes a change in the system.")
+AnnotationAssertion(rdfs:label obo:RO_0007000 "has_driver")
+SubObjectPropertyOf(obo:RO_0007000 obo:RO_0003302)
```

Comment:
@cmungall please ensure that this subproperty assertion makes sense.

Reply:
@cmungall Hello Chris, can you take a look at these definitions, would be great to get these integrated into RO before the holidays. Lots of data to annotate :)

Cheers,
Lynn

Reply:
@matentzn thanks for catching that, sorry I missed the first time round, indeed that subproperty doesn't make sense since it inverts intended domain and range

@lschriml can you remove this? This will mean driver floats to the root but I think this is fine

Reply:
@cmungall @matentzn -- just to make sure I understand and make the correct edits. 

-- do you want me to remove 'has driver' as a sub property of 'causes or contributes to condition' ? 
  
--should 'has driver' instead be a sub property of 'causal relation between two entities' or one step higher to 'causally related to' 

- or do you not want the relationship between 'has driver' and 'has disease driver' ? 

I am fine with your moving it to where it is most appropriate, if that is more expedient. 
Also, happy to make any edits.

Cheers,
Lynn

Reply:
All good, removing it was correct